docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -1897,3 +1897,300 @@ pip install gremlinpython
 pip install aiohttp
 python example.py
 ----
+
+[[gremlin-go]]
+== Gremlin-Go
+
+image:gremlin-go.png[width=130,float=right] Apache TinkerPop's Gremlin-Go implements Gremlin within the link:https://go.dev/[Go] language and can therefore be used on different operating systems. Go's syntax has the similar constructs as Java including
+"dot notation" for function chaining (`a.b.c`) and round bracket function arguments (`a(b,c)`). Something unlike Java is that Gremlin-Go requires a 
+`gremlingo` prefix when using the namespace (`a(b())` vs `gremlingo.a(gremlingo.__.b())`). Anyone familiar with Gremlin-Java will be able to work
```

Comment:
`gremlingo.a(gremlingo.__.b())` is a bit misleading since the double-underscore isn't supported except by way of `var` initialization and then you wouldn't use the `gremlingo` prefix, right?